docs/Project-Configuration/Project-Commands.md

```diff
@@ -493,6 +493,7 @@ Notes:
     tasks to the variant). It is a validation error to define a function, task,
     or variant more than once in the JSON document passed to the command _except_
     to specify a variant multiple times in order to append additional tasks to the variant.
+    It will also not re-evaluate task tags when adding tasks to a variant to activate.
```

Comment:
nit: prefer to make this it's own list item

Reply:
Seperated out and linked